internal/v4/api.go

```diff
@@ -815,3 +826,44 @@ func (h *Handler) serveChangesPublished(_ http.Header, r *http.Request) (interfa
 func (h *Handler) serveMacaroon(_ http.Header, _ *http.Request) (interface{}, error) {
 	return h.newMacaroon()
 }
+
+// GET id/promulgate
+// TODO (mhilton): document admin/promulgate endpoint
```

Comment:
That was an old location for it, more importantly it needs documenting, which is now done.